pkgs/misc/urbit/libaes_siv/default.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, sources, cmake, openssl, enableParallelBuilding ? true }:
+
+stdenv.mkDerivation {
+  name = "libaes_siv";
+  version = sources.libaes_siv.rev;
+  src = sources.libaes_siv;
+  patches = [ ./cmakefiles_static.patch ];
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ openssl ];
+
+  cmakeFlags = [
+    "-DBUILD_SHARED_LIBS=OFF"
+  ];
+
+  inherit enableParallelBuilding;
```

Comment:
This is rather useless since cmake sets it already.
```suggestion
```
Also meta is missing again